src/builder/error.rs

```diff
@@ -68,11 +68,11 @@ quick_error! {
             description("can't read directory")
             display("error reading directory: {:?}", errors)
         }
-        /// Can't compile regex
-        Regex(e: regex::Error) {
+        /// Can't read directory for copying
+        PathFilter(errors: Vec<path_filter::FilterError>) {
```

Comment:
What is the list of errors? Why don't we stop on the first error? Is there any cap on the number of errors here?

If there might be an error per file or something, it may consume lots of memory. While I don't think we're too memory-bound, but it's  much better to bail out on first error than to crash on OOM or hang by swapping when lots of errors being accumulated but not printed. Also note, that by default vagga-box has quite small amount of memory.